### PR TITLE
Chat activity

### PR DESCRIPTION
**Pull request by Headswitcher**
Trello card : https://trello.com/c/X2AOxknA/35-chat-layout-activity-frame

### CODE REVIEW

**Review by bakbalazs — COMMENTED** [review]
test

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -16,7 +16,11 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+      
+        <activity android:name=".chat.MainChatMenu"/>
```
Comment (bakbalazs) [inline app/src/main/AndroidManifest.xml:20]: Teszt

File: app/src/main/java/hu/unideb/smartcampus/chat/MainChatMenu.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+public class MainChatMenu extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_chat);
+        ListView chatRoomsList = (ListView) findViewById(R.id.chat_menu_ListView);
+        List names = Arrays.asList("Holkai","Gabai","Atkai","Norkai","Filtián","Balai");
+        // need function to get all the chat rooms available
+        // need function - and brainstorm about chat history, what the xmpp server does
```
Comment (Headswitcher) [inline app/src/main/java/hu/unideb/smartcampus/chat/MainChatMenu.java:28]: teszt

File: app/src/main/java/hu/unideb/smartcampus/chat/MainChatMenu.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+/**
+ * Created by Headswitcher on 2017. 02. 22..
+ */
+
+public class MainChatMenu extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_chat);
```
Comment (bakbalazs) [inline app/src/main/java/hu/unideb/smartcampus/chat/MainChatMenu.java:24]: Teszt

**Pull request merged**